cli/tests/integration_tests.rs

```diff
@@ -1403,6 +1403,11 @@ itest!(_041_info_flag {
   output: "041_info_flag.out",
 });
 
+itest!(_041_2_info_json {
```

Comment:
I could use some advice here on the name.
I added two new tests in this file, they are just json variations of existing tests. I wanted to stay next to the original ones, but the numbers prevent this. Here there was a "042" test already, so for now I just used "041_2" for now.
Should I just put my new tests at the end of the file ? and should I number them or not (they're not all numbered) ?

Reply:
info_json works